### ISSUE TITLE
Components / Countdown timer - text visibility

### ISSUE THREAD

**Issue by yesenia-gtz** [Office-of-Digital-Services/California-State-Web-Template-Website#973]
[Components / Countdown timer](https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/components/countdown-timer.html)

**Recommendation:**
[Text can be resized](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Text_Size) without assistive technology up to the user agent maximum without loss of content and functionality. Standard font size should be no lower than 12pt.
- [x] Bump up countdown text to 1.2rem.

![Screen Shot 2022-12-13 at 1 11 57 PM](https://user-images.githubusercontent.com/98193284/207444499-87b873f2-d30b-4d38-b309-0fb4fddce1fb.png)

**Comment by yesenia-gtz** [comment]
@sharon-rabang needs design review

**Comment by sharon-rabang** [comment]
@kkoryaka Looks like the example's typography needs to get updated to reflect the size contrast that was in Figma: [Countdown timer](https://www.figma.com/file/nVVip6nhCxHQnzPUnVf1bb/V6--State-Web-Template-official?node-id=478%3A9160&t=tKj9y2j4Puk7e1l6-1)

**Comment by pglomboy** [comment]
@kkoryaka a11y issues have been resolved. Good to go!
